src/components/SignUp/TermsOfService.js

```diff
@@ -1,12 +1,42 @@
+import styled from '@emotion/styled';
 import React from 'react';
 
+const Container = styled.div`
+  margin-bottom: 40px;
+`;
+
+const TOSText = styled.div`
+  overflow: scroll;
+  max-height: 300px;
+  overflow: auto;
+  border: 1px solid #1f3947;
```

Comment:
This color is not in the theme. But can we maybe replace it with a similar one that is there?

Reply:
Yeah I'll find one! I wanted to remove all these hex color literals when doing these refactorings, somehow forgot these. Thanks for noticing!